cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -399,6 +399,49 @@ def test_api_v2_jobs_id_admin_partial(self):
         response = self._run_api_v2_jobs_id(self.job.id, self.owner, data)
         self._check_request(response, data)
 
+class JobDataMetaPartialUpdateAPITestCase(APITestCase):
```

Comment:
@ActiveChooN , can we implement these tests in tests/rest_api instead? These files should not be extended anymore.